src/screens/Messages/Conversation/index.tsx

```diff
@@ -31,10 +32,31 @@ type Props = NativeStackScreenProps<
   CommonNavigatorParams,
   'MessagesConversation'
 >
+
+let prevDid: string | undefined
```

Comment:
A bit odd having state outside of React? Shouldn't this be in a ref?

Reply:
It can't be because the whole tree is gettin nuked when see switch accounts. Ran into the same problem with notifications as well.